content/docs-v1/page-content.md

```diff
@@ -24,37 +25,42 @@ You can read all about Markdown syntax and how it is used in Quire in the [*YAML
 
 ```go
-{{</* q-figure id="3.1" */>}}
+{% figure "3.1" %}
 ```
 
-While most Quire shortcodes work like `q-figure` as a single instance, the `q-class` shortcode acts as wrapper around other text and so it appears as a paired opening and closing shortcode. The closing code has a slash `/` preceding the shortcode name, much like you’d find in HTML markup. This example adds the class "alert" to the the phrase "Text goes here", which could be used to facilitate [style customization](/docs-v1/styles-customization/).
+The following shortcodes are currently available in Quire. You’ll find more about them in their respective sections of the guide, as well as in the [shortcodes API reference](/docs-v1/for-developers/#shortcodes-api).
 
-```go
-{{</* q-class "alert" */>}}
-Text goes here
-{{</* /q-class */>}}
-```
+- [`{% bibliography %}`](/docs-v1/citation-bibliographies/): Generates a bibliography from the entries in the project's `bibliography.yaml` file.
+- [`{% cite %}`](/docs-v1/citation-bibliographies/): Adds a linked Author Date citation reference to the text, and an in-text (or hover pop-up) citation. It also adds the citation to a map of cited works, which can then be output as a page-level bibliography on essay and entry type pages.
+- [`{% contributor %}`](/docs-v1/contributors/): Can be used to create a page of contributor biographies, a section of bios for a single page, a simple list of contributors, a byline for a particular page, or other similar outputs.
+- [`{% figure %}`](/docs-v1/figure-images/): Inserts a formatted figure image (including audio and video) and caption using data from the project’s `figures.yaml` file, or from values supplied directly in the shortcode.
+- [`{% figure-group %}`](/docs-v1/figure-images/): Like `{% figure %}`, but with handling for multiple images at once.
 
 {{< q-class "box tip" >}}
-- Quire includes one pre-defined class called "backmatter". This is typically used to wrap bibliographies, appendices, and other related content at the end of an article or page, and will style them to match the default footnote styling.
 
-    ```go
-    {{</* q-class "backmatter" */>}} ...
-    {{</* /q-class */>}}
-    ```
+- There is an additional shortcode that is use to format backmatter. You can use the pairing of `{% backmatter %}` and `{% endbackmatter %}` to wrap bibliographies, appendices, and other related content at the end of an article or page. Whatever falls inside those two shortcodes will match the default footnote styling.
+
+```md
+{% backmatter %}
```

Comment:
Can you test this? You might need line breaks for it to work properly.

```md
{% backmatter %}

## Notes

{% endbackmatter %}
```

Reply:
Ah yes, I just discovered this with Chloe. The line breaks are necessary